packages/components/src/internal/components/entities/constants.ts

```diff
@@ -131,6 +134,8 @@ export const DataClassDataType: EntityDataType = {
     typeListingSchemaQuery: SCHEMAS.EXP_TABLES.DATA_CLASSES,
     listingSchemaQuery: SCHEMAS.EXP_TABLES.DATA,
     instanceSchemaName: SCHEMAS.DATA_CLASSES.SCHEMA,
+    moveActionName: 'moveSources.api',
```

Comment:
does this `DataClassDataType` not need the `instanceKey` prop? maybe that will come later after we support moving of registry items.

Reply:
ah, nevermind. I see it is handled in the other PRs